tests/unit/test_queries.py

```diff
@@ -65,31 +74,36 @@ def setUp(self):
 
     def test_queries(self):
         """tests that the queries lambda correctly returns the search results"""
-        ## Arrange ##
+        # Arrange
         response_true = [
             {
                 "_index": "test_data",
                 "_type": "_doc",
                 "_id": "1",
-                "_score": 0.2876821,
+                "_score": 0.18232156,
                 "_source": {
                     "mission": "imap",
                     "level": "l0",
                     "instrument": "mag",
                     "date": "20230112",
                     "version": "*",
                     "extension": "pkts",
-                },
+                }
             }
         ]
         self.client.send_document(self.document)
         time.sleep(1)
         event = {"queryStringParameters": {"instrument": "mag"}}
 
-        ## Act ##
-        response_out = queries.lambda_handler(event, "")
+        # Act
+        response = queries.lambda_handler(event, "")
+        response_out = json.loads(response['body'])
+
+        for res in response_out:
+            res['_score'] = None
+        for res in response_true:
```

Comment:
You updated this above, but now are overriding it here to be None? So, maybe you can put `None` in the true body above? But, I don't understand why we can't count on a score value in unit tests, so this force-to-None seems a bit suspect to me that we are missing something.

Reply:
_score is not deterministic, meaning it can change based on the state of the Elasticsearch cluster, and therefore it's not something that can be reliably asserted in a unit test.